backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/modelSettings.test.js

```diff
@@ -53,62 +55,71 @@ Legends.checkLegendExists.mockImplementation((teamspace, model, legend) => {
-		[{ params, body: { surveyPoints: null } }, false, 'with null surveyPoints'],
-		[{ params, body: { surveyPoints: [{ position: [1, 2, 3] }] } }, false, 'with surveyPoints with no latLong'],
-		[{ params, body: { surveyPoints: [{ latLong: [1, 2] }] } }, false, 'with surveyPoints with no position'],
-		[{ params, body: { surveyPoints: [{ position: [1, 2, 3, 4], latLong: [1, 2] }] } }, false, 'with surveyPoints with invalid position'],
-		[{ params, body: { surveyPoints: [{ position: [1, 2, 3], latLong: [1, 2, 3] }] } }, false, 'with surveyPoints with invalid latLong'],
-		[{ params, body: { surveyPoints: [{ position: [1, 2, 3], latLong: [1, 2] }] } }, true, 'with valid surveyPoints'],
-		[{ params, body: { angleFromNorth: 'invalid' } }, false, 'with invalid angleFromNorth'],
-		[{ params, body: { angleFromNorth: null } }, false, 'with null angleFromNorth'],
-		[{ params, body: { angleFromNorth: 123 } }, true, 'with valid angleFromNorth'],
-		[{ params, body: { type: null } }, false, 'with null type'],
-		[{ params, body: { type: 'valid' } }, true, 'with valid type'],
-		[{ params, body: { unit: 'invalid' } }, false, 'with invalid unit'],
-		[{ params, body: { unit: null } }, false, 'with null unit'],
-		[{ params, body: { unit: 'mm' } }, true, 'with valid unit'],
-		[{ params, body: { code: 'CODE1!' } }, false, 'with code that has symbols'],
-		[{ params, body: { code: null } }, false, 'with null code'],
-		[{ params, body: { code: 'CODE1' } }, true, 'with valid code'],
-		[{ params, body: { defaultView: 123 } }, false, 'with invalid defaultView'],
-		[{ params, body: { defaultView: null } }, true, 'with null defaultView'],
-		[{ params, body: { defaultView: '9c7a6c50-ee85-11e8-af42-09344c707317' } }, false, 'with defaultView that does not exist'],
-		[{ params, body: { defaultView: '9c7a6c50-ee85-11e8-af42-09344c707317' } }, false, 'with defaultView that does not exist'],
-		[{ params, body: { defaultView: existingViewID } }, true, 'with defaultView that exists'],
-		[{ params, body: { defaultView: existingViewID } }, true, 'with defaultView that exists'],
-		[{ params, body: { defaultLegend: 123 } }, false, 'with invalid defaultLegend'],
-		[{ params, body: { defaultLegend: null } }, true, 'with null defaultLegend'],
-		[{ params, body: { defaultLegend: '9c7a6c50-ee85-11e8-af42-09344c707317' } }, false, 'with defaultLegend that does not exist'],
-		[{ params, body: { defaultLegend: '9c7a6c50-ee85-11e8-af42-09344c707317' } }, false, 'with defaultLegend that does not exist'],
-		[{ params, body: { defaultLegend: existingLegendID } }, true, 'with defaultLegend that exists'],
-		[{ params, body: { defaultLegend: existingLegendID } }, true, 'with defaultLegend that exists'],
-		[{ body: {} }, false, 'with empty params, body'],
-		[{}, false, 'with undefined params, body'],
-	])('Check if req arguments for settings update are valid', (data, shouldPass, desc) => {
+		[{ params, body: { name: null } }, MODEL_TYPES.CONTAINER, false, 'with null name'],
```

Comment:
I think at this point it'll be cleaner if we separate the tests into 3 test groups